### ISSUE TITLE
Unable to keep all duplicate keys

### ISSUE THREAD

**Issue by xtu** [rickyah/ini-parser#69]
Thanks for this useful lib. I have an INI file with duplicate sections and a same key in every duplicate section. I would like to keep all these keys by concatenate all the values with a separator between them.

Here is an example of the INI file I would like to read.

```
[MySection]
File=a.txt
[MySection]
File=b.txt
[MySection]
File=c.txt
[MySection]
File=d.txt
```

I would like the lib to get the values in this way assuming that `; ` is used as the separator.

```
iniData["MySection"]["File"] = a.txt;b.txt;c.txt;d.txt
```

**Comment by rickyah** [comment]
It seems like a really extreme edge case. Why would you like to use multiple duplicate sections and name instead of just setting the File value directly?

```
[MySection]
File = a.txt;b.txt;c.txt;d.txt
```

**Comment by rickyah** [comment]
Closed to follow conversation in #70
